database/database.go

```diff
@@ -2,13 +2,15 @@ package database
 
 import (
 	"database/sql"
+	"errors"
+	"fmt"
 	"os"
 )
 
 func ConnectDB(dbFile string) (*sql.DB, error) {
 	db, err := sql.Open("sqlite3", dbFile)
 	if err != nil {
-		return nil, err
+		return nil, errors.New(fmt.Sprintf("error while openning %s: %w", dbFile, err))
```

Comment:
Нужно обернуть ошибку, а у тебя она создается с нуля
```suggestion
		return nil, fmt.Errorf("error while openning %s: %w", dbFile, err)
```